views/index.erb

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Peeps</title>
+    <style>
```

Comment:
I'd suggest moving the CSS code into an external CSS file, to keep the HTML and CSS code separated and make it easier to update later — you can follow this guidance to load an external CSS file on the page: https://github.com/makersacademy/web-applications/blob/main/pills/loading_css_or_static_files.md